assets/src/components/parts/janus-text-flow/QuillEditor.tsx

```diff
@@ -51,30 +64,44 @@ const getCssForFonts = (fonts: string[]) => {
 const fontStyles = `${getCssForFonts(supportedFonts)}
 /* default normal size */
 .ql-container {
-  font-size: 14px !important;
-}
-.ql-snow .ql-picker.ql-size .ql-picker-item[data-value="10px"]::before {
-  content: 'Smaller (10px)';
-  font-size: 10px !important;
+  font-size: 20px !important;
 }
 .ql-snow .ql-picker.ql-size .ql-picker-item[data-value="12px"]::before {
-  content: 'Small (12px)';
+  content: '12px';
   font-size: 12px !important;
 }
 .ql-snow .ql-picker.ql-size .ql-picker-item[data-value="14px"]::before {
-  content: 'Normal (14px)';
+  content: '14px';
   font-size: 14px !important;
 }
-
 .ql-snow .ql-picker.ql-size .ql-picker-item[data-value="16px"]::before {
-  content: 'Large (16px)';
+  content: '16px';
   font-size: 16px !important;
 }
-
 .ql-snow .ql-picker.ql-size .ql-picker-item[data-value="18px"]::before {
-  content: 'Larger (18px)';
+  content: '18px';
   font-size: 18px !important;
 }
+.ql-snow .ql-picker.ql-size .ql-picker-item[data-value="20px"]::before {
```

Comment:
This will handle the font size of the selected text not getting populated in the font size selection drop down.